{{ cookiecutter.project_name_dashed }}/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,27 +35,19 @@ Generally, **PRs are only reviewed when ALL BOXES are ticked off and all
 automated checks pass** (use the comment section below if you believe that
 your PR is ready to be merged even though not all boxes were ticked off). -->
 
-- \[ \] My code follows the [contributing guidelines][contributing-guidelines]
-  of this project, including, in particular, with regard to any style guidelines
-- \[ \] The title of my PR complies with the
-  [Conventional Commits specification][conv-commits]; in particular, it clearly
-  indicates that a change is a breaking change
-- \[ \] I acknowledge that all my commits will be squashed into a single commit,
+1. [] My code follows the [contributing guidelines][contributing-guidelines]
+  and I acknowledge that all my commits will be squashed into a single commit
   using the PR title as the commit message
-- \[ \] I have performed a self-review of my own code
-- \[ \] I have commented my code in hard-to-understand areas
-- \[ \] I have updated the user-facing documentation to describe any new or
-  changed behavior
-- \[ \] I have added type annotations for all function/class/method interfaces
-  or updated existing ones (only for Python, TypeScript, etc.)
-- \[ \] I have provided appropriate documentation
-  ([Google-style Python docstrings][py-doc-google]) for all
-  packages/modules/functions/classes/methods or updated existing ones
-- \[ \] My changes generate no new warnings
-- \[ \] I have added tests that prove my fix is effective or that my feature
+2. [] I have performed a self-review of my own code and updated the user-facing
+  documentation to describe any new or changed behavior
+3. [] I have commented my code in hard-to-understand areas and provided
+  appropriate documentation ([Google-style Python docstrings][py-doc-google])
+  for all packages/modules/functions/classes/methods or updated existing ones
+4. [] I have added type annotations for all function/class/method interfaces or
+  updated existing ones (only for Python, TypeScript, etc.)
+5. [] I have added tests that prove my fix is effective or that my feature
   works
-- \[ \] New and existing unit tests pass locally with my changes
-- \[ \] I have not reduced the existing code coverage
+6. [] I have not reduced the existing code coverage
```

Comment:
Please add the additional point about code reviews from #51 at the very end. You can actually put that in **bold**